subprojects/plugins/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt

```diff
@@ -243,6 +243,14 @@ set -- \\
 <% } %>        ${mainClassName} \\
         "\$@"
 
+# Stop when "xargs" is not available.
+
+command -v xargs > /dev/null 2>&1
```

Comment:
:heavy_check_mark:  Should work just fine, but I've also found out that this can be simply done by `&> /dev/null` according to the [Bash manpage](https://stackoverflow.com/a/11255498).

Reply:
I hate to say it but we cannot assume bash here (i.e. the first line is `#!/bin/sh`).

Reply:
Ah... totally forgot that we are in `sh` domain here... Nevermind :laughing: